systemtests/scripts/run_test_component.sh

```diff
@@ -22,6 +22,8 @@ get_kubernetes_info ${LOG_DIR} services default "-before"
 get_kubernetes_info ${LOG_DIR} pods default "-before"
 
 ${CURDIR}/system-stats.sh > ${ARTIFACTS_DIR}/system-resources.log &
+${CURDIR}/docker-logs.sh > ${ARTIFACTS_DIR}/docker-logs &
+
 STATS_PID=$!
```

Comment:
this command (STATS_PID=$!) should be between system-stats.sh and docker-logs.sh and new one should be added for getting PID of docker-logs.sh script 

both background running processes should be killed after systemtests run is done

Reply:
@tkratky Should be fixed now.